src/collectHeatmapClicks.js

```diff
@@ -16,6 +16,19 @@ export default (subscribe, { interval = 100 }) => {
     }
   };
 
+  const performOnScriptEnd = function(fnc) {
```

Comment:
So you want to run it asynchronously, after the shortest possible delay, right?

Reply:
Right

Reply:
However in practice it takes ~1s to run that code. I inspected ContainerProvider code to verify our native PPAS solution including delay and it looks like implementation is similar. Didn't check how it works in practice.